src/main/java/com/github/lombrozo/jsmith/antlr/rules/Identifier.java

```diff
@@ -46,6 +98,16 @@ public String name() {
 
     @Override
     public Rule copy() {
-        return new Identifier(this.parent());
+        return new Identifier(this.parent(), this.ref);
+    }
+
+    /**
+     * Checks if rule is Identifier.
+     * @param rule Rule to check.
+     * @return True if rule is identifier, false otherwise.
+     */
+    static boolean isIdentifier(final Rule rule) {
```

Comment:
@RechnoyBobr It seems that we don't use this method in the code, or I'm missing something? Can we just remove it if it's true?